README.md

```diff
@@ -15,6 +15,7 @@
 | GEOGRAPHY_ENABLED                | false                                     | Geography feature toggle                                                                 |
 | FEEDBACK_CONTROLLER_URL          | <http://localhost:25200>                  | The URL of dp-frontend-feedback-controller                                               |
 | SEARCH_CONTROLLER_URL            | <http://localhost:25000>                  | The URL of dp-frontend-search-controller                                                 |
+| NEW_DATA_AGGREGATION_PAGES_ENABLED   | true                                     | Enables the new data aggregation pages                                                             |
```

Comment:
Let's omit the 'NEW' on this - one day it won't be :D

Reply:
Resolved but realistically I would imagine the whole flag would be removed once the transition to the updated pages is made